doc/troubleshooting.md

```diff
@@ -582,6 +582,32 @@ The name must not be blank.
 
 After correcting the configuration, the [changes will need to be propagated to the Yaskawa controller](../README.md#updating-the-configuration).
 
+### Alarm: 8011[23 - 54]
+
+*Example:*
+
+```text
+ALARM 8011
+ Failed creating/adding/init *something*
+[23] - [54]
+```
+
+*Solution:*
+Ensure that the ROS2 version (Galactic, Humble, etc) is the same for MotoROS2, the micro-ROS Agent and the client PC.
+Additionally, ensure that the ROS2 version is one that is [officially supported](https://github.com/Yaskawa-Global/motoros2#general-requirements) by MotoROS2.
+
+If the behavior persists, save a copy of the [debug-listener script](#debug-log-client) output and the `PANELBOX.LOG` from the robot's teach pendant.
+Open a new issue on the [Issue tracker](https://github.com/yaskawa-global/motoros2/issues).
+Describe the problem and include the following items:
+
+- `PANELBOX.LOG`
+- output from debug listener script (complete and unedited)
+- version of MotoROS2
+- version of micro-ROS Agent
+- version of ROS2 on client PC
+- copy of the MotoROS2 yaml configuration file
```

Comment:
```suggestion
- copy of `motoros2_config.yaml` copied from the robot controller
```

we have some text in the readme detailing how to do this, but it's not in its own section, so we can't directly link to it.